x-pack/legacy/plugins/snapshot_restore/public/app/components/policy_form/steps/step_settings.tsx

```diff
@@ -78,216 +79,239 @@ export const PolicyStepSettings: React.FunctionComponent<StepProps> = ({
+        onChange={e => {
+          const isChecked = e.target.checked;
+          setIsAllIndices(isChecked);
+          if (isChecked) {
+            updatePolicyConfig({ indices: undefined });
+          } else {
+            updatePolicyConfig({
+              indices:
+                selectIndicesMode === 'custom'
+                  ? indexPatterns.join(',')
+                  : [...(indicesSelection || [])],
+            });
+          }
+        }}
+      />
+    );
+
+    return (
+      <EuiDescribedFormGroup
+        title={
+          <EuiTitle size="s">
+            <h3>
               <FormattedMessage
-                id="xpack.snapshotRestore.policyForm.stepSettings.allIndicesLabel"
-                defaultMessage="All indices, including system indices"
+                id="xpack.snapshotRestore.policyForm.stepSettings.indicesTitle"
+                defaultMessage="Indices"
               />
-            }
-            checked={isAllIndices}
-            data-test-subj="allIndicesToggle"
-            onChange={e => {
-              const isChecked = e.target.checked;
-              setIsAllIndices(isChecked);
-              if (isChecked) {
-                updatePolicyConfig({ indices: undefined });
-              } else {
-                updatePolicyConfig({
-                  indices:
-                    selectIndicesMode === 'custom'
-                      ? indexPatterns.join(',')
-                      : [...(indicesSelection || [])],
-                });
-              }
-            }}
+            </h3>
+          </EuiTitle>
+        }
+        description={
+          <FormattedMessage
+            id="xpack.snapshotRestore.policyForm.stepSettings.indicesDescription"
+            defaultMessage="Indices to back up."
           />
-          {isAllIndices ? null : (
-            <Fragment>
-              <EuiSpacer size="m" />
-              <EuiFormRow
-                className="snapshotRestore__policyForm__stepSettings__indicesFieldWrapper"
-                label={
-                  selectIndicesMode === 'list' ? (
-                    <EuiFlexGroup justifyContent="spaceBetween">
-                      <EuiFlexItem grow={false}>
-                        <FormattedMessage
-                          id="xpack.snapshotRestore.policyForm.stepSettings.selectIndicesLabel"
-                          defaultMessage="Select indices"
-                        />
-                      </EuiFlexItem>
-                      <EuiFlexItem grow={false}>
-                        <EuiLink
-                          onClick={() => {
-                            setSelectIndicesMode('custom');
-                            updatePolicyConfig({ indices: indexPatterns.join(',') });
-                          }}
-                        >
+        }
+        idAria="indicesDescription"
+        fullWidth
+      >
+        <EuiFormRow hasEmptyLabelSpace fullWidth describedByIds={['indicesDescription']}>
+          <Fragment>
```

Comment:
Doesn't look like `Fragment` is needed here (same for points below)? I see it was pre-existing.

Reply:
This actually is needed. `<EuiFormRow>` expects a single child.